src/core/financial.jl

```diff
@@ -159,35 +159,35 @@ mutable struct Financial
         onsite_costs = easiur_costs(latitude, longitude, "onsite")
         escalation_rates = easiur_escalation_rates(latitude, longitude, om_cost_escalation_pct)
         if !isnothing(grid_costs)
-            if ismissing(NOx_grid_cost_per_tonne)
+            if NOx_grid_cost_per_tonne == 0.0
```

Comment:
Due to changes that @rathod-b has made since this commit, the Financial struct can indeed be made immutable, however the health emissions cost arguments in its constructor cannot have 0 as their default. With the change you've made here, if a user provides a value of 0 for any of the health emissions cost inputs then it will be overwritten. The default needs to be nothing so that we use EASIUR data only if the user didn't provide values. 
I am going to undo the changes made in this commit, except that I will now use nothing instead of missing and I will also leave the changes to the struct field types because handling of failed EASIUR lookup is now done in the Financial constructor (see https://github.com/NREL/REopt.jl/pull/28#discussion_r903296829).

Reply:
Sounds good, resolving.